rust-keybroker/keybroker-server/src/main.rs

```diff
@@ -198,9 +202,13 @@ struct Args {
     endpoint: Option<String>,
 
     /// The URL where the verifier can be reached
-    #[arg(long, default_value = "http://veraison.test.linaro.org:8080")]
+    #[arg(long, default_value = "https://veraison.test.linaro.org:8080")]
     verifier: String,
 
+    /// The verifier root certificate to use for a TLS connection
+    #[arg(long, default_value = "certs/veraison-root.crt")]
```

Comment:
Even though we make it slightly more painful for the user, I would avoid a default here for two reasons:

1. The local trust anchor may already be configured with the right cert (for example, if Veraison gets its cert from LetsEncrypt or another trusted root CA);
2. The connection may be over HTTP and needs no cert-related settings.

Reply:
Agreed, this was basically / links to my questions in the comment at the top of this pull request

Reply:
Addressed in the next iteration of the patch.